apps/android/README.md

```diff
@@ -17,12 +17,28 @@ yarn
 yarn build
 ```
 
-2. Then go into `apps/android` folder and simply run the the following
+2. To build the bundle run:
 
 ```sh
+cd apps/android
+yarn bundle
```

Comment:
Do you actually have to bundle AND run the packager? I remember Android really didn't like my having a bundle so I guess it makes sense.

Reply:
Hmm I'll try it. I think I just run bundle out of habit

Reply:
Seems bundle isn't needed. I'll take it out.